Tests/AutoTests/AccessebilityTest/accessebility.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect, chromium, Page, Browser, BrowserContext } from '@playwright/test';
+import * as data from "../../configuration/pageObject.json";
+import * as app from "../../Configuration/appConfig.json";
+import headerPage from '../../pageObject/home.page';
+import { injectAxe, getViolations, checkA11y} from 'axe-playwright'
+
+test.describe("Goto maritime-safety-information Home Page ", ()=> {
+  let browser: Browser;
```

Comment:
You don't need to re-define Browser, BrowserContext & Page variables as these are already created by the framework and can be passed into the tests as parameters as per our template project https://github.com/UKHO/playwright-template/blob/main/tests/a11y/a11y.spec.ts

Reply:
resolved for the browser